pkg/operator/revisioncontroller/revision_controller.go

```diff
@@ -366,6 +366,21 @@ func (c RevisionController) sync(ctx context.Context, syncCtx factory.SyncContex
 		return nil
 	}
 
+	// Make sure that revision-ready is set to true on the
+	// revision-status{status.latestAvailableRevision} configmap. This prevents
+	// blocking the controller when upgrading to a version that checks revision
+	// readiness.
```

Comment:
I think this is correct. The other controllers in the revision system don't care about the revision-status configmaps, so if latestAvailableRevision has already been updated then the revision has already "shipped" and might be installed.